Code/MinimalLib/README.md

```diff
@@ -85,60 +92,26 @@ To use RDKit, load the javascript file and instantiate the wasm module inside th
 
 ```
 
-## Usage
-
-See the getting started demo at https://unpkg.com/@rdkit/rdkit/Code/MinimalLib/dist/GettingStartedInJS.html .
-
-Follow the examples of this page to see the various ways to use the JavaScript release of RDKit.
-
 ## Live demos
 
-- From this npm package: https://unpkg.com/@rdkit/rdkit/Code/MinimalLib/dist/GettingStartedInJS.html
-- From this npm package: https://unpkg.com/@rdkit/rdkit/Code/MinimalLib/dist/demo.html
+If you are using the MinimalLib for the first time, see the getting started examples at https://www.rdkitjs.com/ .
 
-## Contributing
+### All live demos
 
-### Preparing a new release of the package
+- RDKit.js website: https://www.rdkitjs.com/
+- RDKit.js usage with React.js: https://react.rdkitjs.com/
```

Comment:
Ditto

Reply:
there are two different links here, one for vanilla JS examples https://www.rdkitjs.com/ , and one for React examples https://react.rdkitjs.com/

Reply:
that's my dream @ptosco :smile:  , see a related comment in your repository here https://github.com/rdkit/rdkit-structure-renderer/pull/2#issuecomment-1179290487

maybe we should open an official discussion topic to establish a plan on how to centralize all rdkit-js efforts?